### PR TITLE
remove alpha dependency

### PR DESCRIPTION
**Pull request by m-nash**
# Description
<i>Add your description here!</i>

# Checklist

To ensure a quick review and merge, please ensure:
- [ ] The PR has a understandable title and description explaining the _why_ and _what_.
- [ ] The PR is opened in draft if not ready for review yet.
   - If opened in draft, please allocate sufficient time (24 hours) after moving out of draft for review
- [ ] The branch is recent enough to not have merge conflicts upon creation.

# Ready to Land?
- [ ] Build is completely green
   - Submissions with test failures require tracking issue and approval of a CODEOWNER
- [ ] At least one +1 review by a CODEOWNER
- [ ] All -1 reviews are confirmed resolved by the reviewer 
   - Override/Marking reviews stale must be discussed with CODEOWNERS first

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,7891 +1,7 @@
 {
   "name": "@autorest/csharp",
```
Comment (pakrym) [inline package-lock.json:2]: unrelated?

**Review by pakrym — APPROVED** [review]

File: src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpProj.cs [hunk 2]
```diff
@@ -114,7 +109,7 @@ public async Task<bool> Execute(IPluginCommunication autoRest)
 
                 if (configuration.AzureArm)
                 {
-                    csProjContent = string.Format(_csProjContent, additionalContent, _armCoreCsProjContent);
+                    csProjContent = string.Format(_csProjContent, additionalContent, _coreCsProjContent);
                 }
                 else
                 {
```
Comment (ShivangiReja) [inline src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpProj.cs:115]: We don't need this if, else condition, now that we are using the same core version. Code under the if, else is exactly the same, we can remove the if, else now.

**Review by ShivangiReja — APPROVED** [review]
